turbinia/config/turbinia_config.py

```diff
@@ -16,20 +16,57 @@
 from __future__ import unicode_literals
 
 
-"""Dummy Turbinia config file."""
+"""Turbinia Config Template"""
 
-# Turbinia Config
+##########
+# COMMON #
+##########
 
-# 'PSQ' is currently the only valid option
+# Which user account Turbinia runs as
+USER = 'turbinia'
+
+# Turbinia's installion directory
+TURBINIA_DIR = '/opt/turbinia'
+
+# Turbinia source code directory (local git repository)
+SRC_DIR = '%s/src' % TURBINIA_DIR
+
+# GCSFuse mount for Turbinia scripts & logging and Google Cloud Functions
+MOUNT_POINT = '/mnt/turbinia'
+
+# Turbinia CLI
+TURBINIACTL = '%s/turbinia/turbiniactl' % SRC_DIR
+
+# Virtualenv directory
+TURBINIAENV = '%s/turbinia-env' % HOME_DIR
+
+# Virtualenv activator
+VIRTUALENV_ACTIVATE = '%s/bin/activate' % TURBINIAENV
+
+# GCS bucket that has Turbinia-specific scripts and
+# that can be used to store logs.
+BUCKET = 'turbinia'
+
+# Local directory for temporary data
+TMP_DIR = '/var/tmp'
+
+############
+# TURBINIA #
+############
+
+# 'PSQ' is currently the only valid option as
+# a distributed task queue using Google Cloud Pub/Sub
 TASK_MANAGER = 'PSQ'
 
-# File to log to
-LOG_FILE = None
+# File to log to; set this as None if log file is not desired
+# By default, Turbinia logs are written to a directory (MOUNT_POINT) in the GCS mount
+LOG_FILE = '%s/output/logs/turbinia.log' % MOUNT_POINT
 
-# Default output directory
-OUTPUT_DIR = None
+# Default base output directory for worker results and evidences
+# For local, you can set this to, for example, %/turbinia_output' % TMP_DIR
```

Comment:
Maybe:
s/For local/When running Turbinia locally'

Reply:
Done